bot/seasons/halloween/8ball.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'halloween', 'responses.json'), 'r', encoding="utf8") as f:
+    responses = json.load(f)
+
+
+class EightBall(commands.Cog):
```

Comment:
```suggestion
class SpookyEightBall(commands.Cog):
```

Minor tweak so there's less confusion between loading this vs. loading [evergreen's implementation](https://github.com/python-discord/seasonalbot/blob/master/bot/seasons/evergreen/magic_8ball.py).